frontend/src/components/print/print-react/components/tableOfContents/entry/Program.jsx

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line no-unused-vars
+import React from 'react'
+import { Link, Text } from '@react-pdf/renderer'
+import tocStyles from '../tocStyles.js'
+import sortBy from 'lodash/sortBy.js'
+
+function Program(props) {
+  const periods = props.entry.options.periods.map((periodUri) =>
+    props.store.get(periodUri)
+  )
+  if (!periods.some((period) => period.scheduleEntries().items.length > 0)) {
+    return <React.Fragment />
+  }
+  return periods.map((period) => {
+    const periodEntry = (
+      <Link style={tocStyles.entry} href={`#${props.id}-${period.id}`}>
```

Comment:
Found the problem. It's because the target id is defined on a `<View>` which is split across multiple pages. Apparently each of these split instances gets a copy of the same id... Probably a bug. But I worked around it by adding an empty `<View>` at the top of the program and moved the id and bookmark onto there.